packages/math/init.lua

```diff
@@ -344,6 +359,18 @@ Finally, here is a little secret. This notation:
 
 \noindent In other words, the notation using \code{&} and \code{\\\\} is only a syntactic sugar for a two-dimensional array constructed with braces.
 
+\paragraph{Numbered equations}
+Equations can be numbered in display mode adding the options \autodoc:parameter{numbered} and \autodoc:parameter{counter}.
+When \autodoc:parameter{numbered=true}, the equations are numbered using a default "equation" counter.
+A different counter may be set by using the option \autodoc:parameter{counter=id}, and this setting will also enable numbering. 
```

Comment:
Suggestion: `\autodoc:parameter{counter=id}` --> `\autodoc:parameter{counter=<id>}`
The angle brackets are replaced with nice brackets and the content is shown in a different color when `autodoc.autodoc.highlighting` is enabled. (And italic would possibly be used, although maybe the Hack Italic font is not installed in regular builds, IIRC)